src/cluster.cc

```diff
@@ -531,7 +531,7 @@ Status Cluster::ParseClusterNodes(const std::string &nodes_str, ClusterNodes *no
     std::string host = fields[1];
 
     // 3) port
-    auto parse_result = ParseInt<int>(fields[2], NumericRange<int>{1, 65535 - kClusterPortIncr - 1}, 10);
+    auto parse_result = ParseInt<int>(fields[2], 10);
```

Comment:
I think `ParseInt<uint16_t>` is more concrete here.

Reply:
```suggestion
    auto parse_result = ParseInt<uint16_t>(fields[2], 10);
```